confgenerator/otel/modular.go

```diff
@@ -24,14 +24,36 @@ import (
 
 const MetricsPort = 20201
 
+type ExporterType int
+
+const (
+	// N.B. Every ExporterType increases the QPS and thus quota
```

Comment:
Just curious: Why would new exporter type increase QPS?

Reply:
Yeah, it's only the case if we're not filling up batches. But the batch size is 200 points, so I think in most applications we're usually sending partial batches.